src/mjolnir/directededgebuilder.cc

```diff
@@ -26,7 +26,7 @@ DirectedEdgeBuilder::DirectedEdgeBuilder(const OSMWay& way,
                                          const bool minor,
                                          const uint32_t restrictions,
                                          const uint32_t bike_network,
-                                         const bool reclass_ferry)
+                                         const bool remove_destonly)
```

Comment:
the whole change in this file is only renaming the arg and update the comment, no logic changed